benchexec/tools/nacpa.py

```diff
@@ -0,0 +1,92 @@
+    """
+    This tool-info module runs nacpa, a parallel portfolio
+    of natively compiled CPAchecker instances.
+    """
+
+    REQUIRED_PATHS = ["bin", "lib"]
+
+    def executable(self, tool_locator):
+        return tool_locator.find_executable("nacpa", subdir="bin")
+
+    def version(self, executable):
+        return self._version_from_tool(
+            executable, "--version", line_prefix="nacpa version"
+        )
+
+    def program_files(self, executable):
+        return self._program_files_from_executable(
+            executable, self.REQUIRED_PATHS, parent_dir=True
+        )
+
+    def name(self):
+        return "nacpa"
+
+    def cmdline(self, executable, options, task, rlimits):
+        if "--spec" not in options and task.property_file:
+            options = options + ["--spec", task.property_file]
+
+        if (
+            "--data-model" not in options
+            and isinstance(task.options, dict)
+            and task.options.get("language") == "C"
+        ):
+            data_model = task.options.get("data_model")
+            if data_model:
+                options = options + ["--data-model", data_model]
+
+        return super().cmdline(executable, options, task, rlimits)
+
+    def determine_result(self, run):
+        for line in reversed(run.output):
+            if line.startswith("VERDICT: "):
+                verdict = line.partition(":")[-1].strip().lower()
+                if verdict.lower().startswith("true"):
+                    return result.RESULT_TRUE_PROP
+                elif verdict.lower().startswith("false(unreach-call)"):
+                    return result.RESULT_FALSE_REACH
+                elif verdict.lower().startswith("false(termination)"):
+                    return result.RESULT_FALSE_TERMINATION
+                elif verdict.lower().startswith("false(no-overflow)"):
+                    return result.RESULT_FALSE_OVERFLOW
+                elif verdict.lower().startswith("false(no-deadlock)"):
+                    return result.RESULT_FALSE_DEADLOCK
+                elif verdict.lower().startswith("false(valid-deref)"):
+                    return result.RESULT_FALSE_DEREF
+                elif verdict.lower().startswith("false(valid-free)"):
+                    return result.RESULT_FALSE_FREE
+                elif verdict.lower().startswith("false(valid-memtrack)"):
+                    return result.RESULT_FALSE_MEMTRACK
+                elif verdict.lower().startswith("false(valid-memcleanup)"):
+                    return result.RESULT_FALSE_MEMCLEANUP
+                elif verdict.lower().startswith("false"):
+                    return result.RESULT_FALSE_PROP
+        return result.RESULT_UNKNOWN
+
+    def get_value_from_output(self, output, identifier):
+        # search for the text in output and get its value,
+        # search the first line, that starts with the searched text
+        # warn if there are more lines (multiple statistics from sequential analysis?)
```

Comment:
Hm, I did not find any code that corresponds to this claim. Was this overlooked or is the comment wrong?

Reply:
Removed.